src/features/hotspots/setup/HotspotSelectionScreen.tsx

```diff
@@ -109,6 +111,89 @@ const HotspotSetupSelectionScreen = () => {
           </TouchableHighlightBox>
         </Card>
       </Box>
+
```

Comment:
We probably need a scrollview surrounding these cards too, not sure that they will fit on small devices?